xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,114 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+SELECT_FIRST_UNWATCHED_ITEM CGUIWindowVideoNav::SelectFirstUnwatchedItem()
```

Comment:
Have prefixed the enum names with e - so eSelectFirstUnwatchedItem - hope that is acceptable.